tests/trainers/java/time.java

```diff
@@ -0,0 +1,68 @@
+/**
+Test to check the Java Timming. This cover only the adquisition
+of the current date in different ways (because is dependent of
+the system ), but not all the operations can be performed
+- Date
+- LocalTimeDate
+- currentTimeMillis
+- nanoTime
+- instant
+*/
+import java.util.*;
+import java.time.*;
+
+public class Test {
+    void date() throws Exception, InterruptedException {
+        Date d1 = new Date();
+        Thread.sleep(100);
+        Date d2 = new Date();
+        if (d2.getTime() <= (d1.getTime() + 100))
+            throw new Exception("java.util.Date");
+    }
```

Comment:
I would suggest to include the actual values in the exception text so that you know _why_ it failed and not only _that_ it failed.
Also, why don't you use an `assert` for this? Doesn't that throw an exception as well?

Reply:
Yes I test this with assert and works perfectly well. Thank you